pom.xml

```diff
@@ -503,5 +535,44 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>benchmark</id>
+      <properties>
+        <skipTests>true</skipTests>
+        <benchmark>org.apache</benchmark>
+      </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>exec-maven-plugin</artifactId>
+            <version>3.0.0</version>
+            <executions>
+              <execution>
+                <id>benchmark</id>
```

Comment:
Strange, we already have a `benchmark` Maven profile. Did you rebase when you added your JMH changes @ali-ghanbari ? https://github.com/apache/commons-text/blob/c4b2c2774b3d797df74a87418c4818aa6d0a8538/pom.xml#L538

Reply:
Yes @kinow, we do have a JMH benchmark profile already in the latest version in `master` branch.
I actually copied from there to reduce the risk of conflicts.

> Did you rebase when you added your JMH changes?

I just manually copied relevant parts into `pom.xml`.

Reply:
@kinow :
And since we already have a JMH benchmark, I suggest keeping LCS's benchmark within a separate package. Is that necessary?